Patch to remove (now) erroneous entries from the manifest used
to create the egg. Not suggested upstream since this is specific
to our userland build system.

--- cx_Oracle-6.1/MANIFEST.in.orig	2017-12-13 16:23:18.269036782 -0800
+++ cx_Oracle-6.1/MANIFEST.in	2017-12-13 16:23:25.011454707 -0800
@@ -1,9 +1,5 @@
 include MANIFEST.in
 include *.txt
-recursive-include odpi *.c
-recursive-include odpi *.h
-prune odpi/test
-prune odpi/samples
 recursive-include src *.c
 recursive-include samples *.py *.sql
 recursive-include test *.py *.sql
